server/chat-commands.js

```diff
@@ -798,7 +798,7 @@ const commands = {
 
 		Ladders.visualizeAll(target).then(values => {
 			let buffer = `<div class="ladder"><table>`;
-			buffer += `<tr><td colspan="8">User: <strong>${Chat.escapeHTML(target)}</strong></td></tr>`;
+			buffer += Chat.html`<tr><td colspan="8">User: <strong>${target}</strong></td></tr>`;
```

Comment:
Won't this cause real HTML to get escaped?

Reply:
There's no real HTML in here...